plugins/providers/metabase/provider.go

```diff
@@ -62,10 +62,21 @@ func (p *provider) GetResources(pc *domain.ProviderConfig) ([]*domain.Resource,
 			db.ProviderType = pc.Type
 			db.ProviderURN = pc.URN
 			resources = append(resources, db)
+
+			if _, ok := resourceTypes[ResourceTypeTable]; ok && resourceTypes[ResourceTypeTable] {
+				tables := d.Tables
+				for _, t := range tables {
+					t.Database = db
+					table := t.ToDomain()
+					table.ProviderType = pc.Type
+					table.ProviderURN = pc.URN
+					resources = append(resources, table)
+				}
+			}
 		}
 	}
 
-	if containsString(resourceTypes, ResourceTypeCollection) {
+	if _, ok := resourceTypes[ResourceTypeCollection]; ok && resourceTypes[ResourceTypeCollection] {
```

Comment:
same here also.

Reply:
I am checking the value should be true in the map